atomic_reactor/plugins/pre_stop_autorebuild_if_disabled.py

```diff
@@ -52,13 +52,16 @@ def _is_rebuild_enabled(self):
                               'enabled' if result else 'disabled',
                               self.config_file)
```

Comment:
Where is result set to `True`? More importantly, if it is never set to `True`, how come the unit testing doesn't catch that?